docs/src/examples/components/Input/Variations/InputExampleIconChange.shorthand.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Input, Icon } from '@stardust-ui/react'
+
+class InputExampleIconChangeShorthand extends React.Component<
+  {},
+  { icon: string; inputValue: string }
+> {
```

Comment:
No typings in examples, please.  The public facing code is not typescript but JS.  We want to advertise the most minimal syntax possible to users in order to focus on our features.  We also don't want to assume they are using typescript since this is a small percentage of the community.